Nexus.Sdk.Shared/Nexus.Sdk.Shared.csproj

```diff
@@ -18,6 +18,7 @@
 
   <ItemGroup>
     <PackageReference Include="IdentityModel" Version="7.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore" Version="8.0.4" />
```

Comment:
Is this package necessary? If not please remove

Reply:
Yes, it's for using Microsoft.AspNetCore.Diagnostics;
Needed to create a custom IExceptionHandler